docs/units/equivalencies.rst

```diff
@@ -111,8 +111,7 @@ energy can be converted.
 These equivalencies even work with non-base units::
 
   >>> # Inches to calories
-  >>> from astropy.units import imperial
-  >>> imperial.inch.to(imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
+  >>> u.imperial.inch.to(u.imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
```

Comment:
It's only necessary when "enabling" the units, which is not the same as making them available (and again was part of the confusion I think):

```
from astropy.units import imperial
imperial.enable()
```